README.md

```diff
@@ -12,8 +12,11 @@ Astarte ESP32 SDK lets you connect your ESP32 device to
 The SDK simplifies all the low-level operations like credentials generation, pairing and so on, and
 exposes a high-level API to publish data from your device.
 
-Have a look at the [examples](examples/README.md) for a usage example showing how to send and
-receive data.
+Check out the examples on the right pane of the
```

Comment:
```suggestion
Check out the examples on the right panel of the
```

Reply:
I copied and pasted from wolfssl's documentation this sentence. I think they refer to a pane on purpose.